.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = tab
```

Comment:
I prefer using spaces for indentation. What is the motivation behind this change?